src/librustdoc/html/static/main.js

```diff
@@ -2113,8 +2113,12 @@ function defocusSearchBar() {
         }
     }
 
+    function getToggleAllDocsElement() {
+        return document.getElementById("toggle-all-docs");
+    }
+
     function toggleAllDocs(pageId, fromAutoCollapse) {
-        var innerToggle = document.getElementById("toggle-all-docs");
+        var innerToggle = getToggleAllDocsElement();
```

Comment:
I have two comments.
1. You forgot to use this function on [line 2363](https://github.com/rust-lang/rust/pull/76022/files#diff-f3aa05ee448be83e65a501258d3aac2bR2363).
2. I'm not sure how much of an improvement this is. It seems a little silly to have a one-line function like this - if it's only used in one place there's no point, and if it's used in multiple places it seems like it's introducing indirection for no reason, you now have to go and look what the function does . I would prefer to have `const toggleAllDocs = "toggle-all-docs"` or `const toggleAllDocsElement = document.getElementById("toggle-all-docs")`, which at least make it clear there's nothing funny going on under the hood.

Reply:
1. Good catch, thanks!
2. This is debatable: the point of doing this is to centralize such operations but also to "hide" its inner operations, explanations: we replace the whole `document.getElementById("...")` boilerplate by a function name which is explicit enough so that you know what it returns, meaning you don't need to take a look at the function, unless you have the need for it. But in the end, if we use a const, you get the same issue: you need to go to the const definition if you want the value.

Reply:
>  the point of doing this is to centralize such operations but also to "hide" its inner operations, 

Right, my point is that I don't think hiding it is helpful. It's doing something very simple, so the operations are easy to read, and I think it's more clear to just have it at the call site. The use case I _do_ see is avoiding typos, which would be better serve by constants.

Reply:
Hum... I'm not a big fan of constants but in this case, I guess it's acceptable. I'll update the code.